doc/quickstart.md

```diff
@@ -59,7 +59,7 @@ On CentOS and openSuSE:
     yum install -y ninja-build meson libyaml-devel
     # Required SPDK packages for managing NVMe SSDs
     yum install -y CUnit-devel libaio-devel python-pep8 lcov
-    yum install -y python clang-analyzer sg3_utils libiscsi-devel
+    yum install -y python python-devel clang-analyzer sg3_utils libiscsi-devel
```

Comment:
mpi4py was removed yesterday #269  so this is no longer needed

Reply:
I'll get rid of adding python-dev. In Ubuntu section below, python2.7-dev is listed. Is it needed if mpi4py is removed, or is it there for another reason?

Reply:
I don’t think it is For DAOS.  IOF and CPPR (both sort of deprecated) use it.